dragonflybsd/DragonFlyBSDProcessList.h

```diff
@@ -8,13 +8,16 @@ Released under the GNU GPLv2, see the COPYING file
 in the source distribution for its full text.
 */
 
+#include <sys/types.h>  // required for kvm.h
 #include <kvm.h>
 #include <sys/param.h>
 #include <osreldate.h>
 #include <sys/jail.h>
 #include <sys/uio.h>
 #include <sys/resource.h>
+
 #include "Hashtable.h"
+#include "ProcessList.h"
```

Comment:
Should also include `#include "Usertable.h"` based on signature of `ProcessList_new`. Also missing `stdbool.h`.

Reply:
Should be addressed by htop-dev/htop#635